### PR TITLE
fix(@angular-devkit/build-angular): use a dot in chunk names

### PR DESCRIPTION
**Pull request by cexbrayat**
## PR Checklist

Please check to confirm your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular-cli/blob/main/CONTRIBUTING.md#-commit-message-guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

The esbuild based builder generates chunks named `chunk-xxx.js`.

## What is the new behavior?

This updates the esbuild based builder to use `chunk.xxx.js` instead of `chunk-xxx.js` to be consistent with the other generated files (`main.xxx.js`, `polyfills.xxx.js`, etc).

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/angular_devkit/build_angular/src/builders/jest/index.ts [hunk 1]
```diff
@@ -109,7 +109,7 @@ export default createBuilder(
       `--testPathIgnorePatterns="<rootDir>/init-test-bed\\.mjs"`,
 
       // Skip shared chunks, as they are not entry points to tests.
-      `--testPathIgnorePatterns="<rootDir>/chunk-.*\\.mjs"`,
+      `--testPathIgnorePatterns="<rootDir>/chunk\\..*\\.mjs"`,
```
Comment (cexbrayat) [inline packages/angular_devkit/build_angular/src/builders/jest/index.ts:112]: I'm not sure if this one is related.

**Comment by clydin** [comment]
I think it may actually be better to make the opposite change.  It's something that was discussed previously for the existing build system. I think the dash provides a clearer signal that the text is not part of the file extension.

File: packages/angular_devkit/build_angular/src/builders/jest/index.ts (hunk 1 above)
Comment (alan-agius4) [reply packages/angular_devkit/build_angular/src/builders/jest/index.ts:112]: It is.

**Comment by cexbrayat** [comment]
Ok no worries, I'll update the PR to use a dash everywhere 👍

**Pull request closed without merging**